sonar-python-plugin/pom.xml

```diff
@@ -150,8 +150,8 @@
             <configuration>
               <rules>
                 <requireFilesSize>
-                  <maxsize>9300000</maxsize>
-                  <minsize>8300000</minsize>
+                  <maxsize>13000000</maxsize>
```

Comment:
I think you forgot to remove the old protobuf files from the previous location. After deleting those, we won't probably need to change jar size constraints.

When we serialize protobuf from `typeshed_serializer`, I think we should add code to clean the existing protobuf files. However , no need to do it in this PR, let's do that in a separate ticket/PR.